pkg/execute/kubectl_cmd_builder.go

```diff
@@ -12,19 +12,23 @@ import (
 	"github.com/slack-go/slack"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/strings/slices"
 
 	"github.com/kubeshop/botkube/pkg/bot/interactive"
 	"github.com/kubeshop/botkube/pkg/config"
 	"github.com/kubeshop/botkube/pkg/execute/kubectl"
 )
```

Comment:
I noticed another problem with not properly resource name cleaned up. Watch this:


https://user-images.githubusercontent.com/7155799/195316607-3a55f1a3-8d0a-4b0f-9299-e5420c26593a.mp4


Also, how to clear the resource name? Can we append an option at the top ("None") or something like this?

(This can be followed-up on different PR)

Reply:
I also spot this bug previously, I will fix it in the next PR. Thanks! I will also talk to Lacho about an option to clear this. Maybe sth like `@None` could be introduced but it still can be misleading  😞 and we are not able to easily clear the state of only one dropdown. I can clear it but for the whole section only 😞